terraform/environment/ai_search.tf

```
@@ -0,0 +1,14 @@
+resource "azurerm_resource_group" "az_search_rg" {
+  name     = "${local.resource_prefix}-rg-search"
```

Again from the "official" MS abbreviations I think this should be:
```suggestion
  name     = "${local.resource_prefix}-rg-srch"
```